rubrik_cdm/cluster.py

```diff
@@ -20,9 +20,118 @@
 
 
 class Cluster(Api):
-    """This class contains methods related to the managment of the Rubrik cluster itself.
+    """This class contains methods related to the management of the Rubrik cluster itself.
     """
 
+    def set_cluster_location(self, location,
+                                   timeout=15):
+        """Configure cluster geolocation. Overwrites previously set value if different.
+
+        Arguments:
+            location {str} -- Geolocation of the cluster.
+
+        Keyword Arguments:
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            str -- If already configured with the same, the location is returned
+            dict -- The full API response from `PATCH /cluster/me`.
+        """
+
+        if not isinstance(location, str):
+            raise InvalidParameterException('The set_cluster_location() function requires the location to be specified as string.')
+
+        self.log("set_cluster_location: Determing the current cluster location.")
+
+        cluster_summary = self.get("v1", "/cluster/me", timeout=timeout)
+
+        if cluster_summary["geolocation"]["address"] == location:
+            return "No change required. The Rubrik cluster is already configured with '{}' as its location.".format(location)
+
+        config = {}
+        config["geolocation"] = {}
+        config["geolocation"]["address"] = location
+
+        self.log("set_cluster_location: Configuring the Rubrik cluster location.")
+
+        return self.patch("v1", "/cluster/me", config, timeout)
+
+    def configure_replication_private(self, username, password, target_ip,
```

Comment:
`def configure_replication_private(self, username, password, target_ip,` same as above. Let's make sure this is a single line.